setup.go

```diff
@@ -110,6 +110,19 @@ func parse(c *caddy.Controller) (*Gateway, error) {
 					return nil, c.Errf("ttl must be in range [0, 3600]: %d", t)
 				}
 				gw.ttlLow = uint32(t)
+			case "negttl":
+				args := c.RemainingArgs()
+				if len(args) == 0 {
+					return nil, c.ArgErr()
+				}
```

Comment:
There is code repetition where each case contains such structure:

```go
args := c.RemainingArgs()
if len(args) == 0 {
	return nil, c.ArgErr()
}
```
do you think we could put it in front of the switch block and have it once ?